assets/app-preact/config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "packageManifest": {
```

Comment:
I like this a lot! Btw this is the first PR on a project of mine. Thanks ;)

1. How should we go about asking the user which template they'd like to use?

I have an idea, but im not sure if theres already a best practice regarding stuff like this.

We could, for example, enable cli flags i.e.
```
npx create-pika-app --template preact
```
This would execute a "silent" install, without any user input necessary.

Running it without any flags would produce a multi-select type menu with the available choices. 

2. Regarding the `package.json` generation. I agree hardcoding the generation / install commands in the primary node binary is not such a good idea.. 

Do we just want to make some generic output, like `installing dependencies`, and `installing dev dependencies` so that these lines dont have to be created manually for each template?